tests/unit/test_session.py

```diff
@@ -66,21 +66,26 @@ def test_credentials_can_be_set(self):
             aws_access_key_id='key',
             aws_secret_access_key='secret',
             aws_session_token='token',
+            aws_account_id='account',
```

Comment:
I'd like to see this maybe split into two tests, one with and one without account id.  We should still be testing that credentials can be set without account id